segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/AttributeId.java

```diff
@@ -233,7 +233,7 @@ public static final class Variable extends AttributeId {
 
         private Variable(byte[] data) {
             this.data = data;
-            this.hashCode = AbstractBufferView.hashCode(this.data);
+            this.hashCode = (int) BufferView.wrap(this.data).hash();
```

Comment:
My understanding is that these are not persisted anywhere durable. If that is not true, then this may be a compatibility problem, otherwise it should be fine.

Reply:
The code before was under the hood making an array copy of the data. (In the library code) This code is just creating a wrapper object which is discarded. So it should be much faster.